v4_MIGRATION_GUIDE.md

```diff
@@ -99,6 +100,86 @@ In v4, we decoupled the Braintree SDK from Android to offer more integration fle
 `BraintreeFragment` has been replaced by a `Client` for each respective payment feature. 
 See the below payment method sections for examples of instantiating and using the feature clients. 
 
+## Authorization Provider
+
+When creating a `BraintreeClient`, you can provide a tokenization key, client token, or an `AuthorizationProvider`. When given an authorization provider, the SDK will fetch a client token on your behalf. This makes it possible to construct a `BraintreeClient` instance in `onCreate`.
+
+**Assuming** the merchant can GET `https://www.my-api.com/client_token` and receive the following json response:
```

Comment:
```suggestion
**Assuming** you have a server that supports GET `https://www.my-api.com/client_token` and receive the following json response:
```

I think we speak directly to merchants in this guide - does this make more sense?